src/index.html

```diff
@@ -11,7 +11,241 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <div class="form-field">
+        <fieldset>
+          <legend>Personal information</legend>
+          <div>
+            <label for="username">
+            Surname:
+            </label>
+
+            <input
+            class="input-class"
+            type="text"
+            name="Username"
+            id="username"
+            required
+             >
+          </div>
+
+          <div>
+            <label for="name">
+            Name:
+            </label>
+            <input
+            class="input-class"
+            type="text"
+            name="Name"
+            id="name"
+            required
+               >
```

Comment:
```suggestion
            <input
              class="input-class"
              type="text"
              name="Name"
              id="name"
              required
             >
```

format code according to checklist